docs/documentation/server_admin/topics/authentication/flows.adoc

```diff
@@ -343,7 +343,7 @@ claims= {
 ----
 
 The {project_name} javascript adapter has support for easy construct of this JSON and sending it in the login request.
-See link:{adapterguide_link_js_adapter}[Javascript adapter documentation] for more details.
+See *Keycloak JavaScript adapter* in the link:https://www.keycloak.org/guides#securing-apps[securing apps] section for more details.
```

Comment:
Same like my other comment (can be hopefully attribute instead of hardcoded link).